docs/eventing/brokers/broker-types/kafka-broker/README.md

```diff
@@ -261,8 +265,46 @@ kubectl create secret --namespace <namespace> generic <my_secret> \
   --from-file=user.key=<my_key.pem_file_path>
 ```
 
-!!! note
-    `ca.crt` can be omitted to fallback to use system's root CA set.
+### Authentication for AWS MSK IAM
+AWS MSK IAM authentication requires creation of a secret and java properties configuration.
+
+In the following ConfigMaps append the following to the listed property values. If using an assumed IAM role, add `awsRoleArn="<role_arn>"` to the `sasl.jaas.config` value.
+
+- config-kafka-broker-data-plane
+  - config-kafka-broker-producer.properties
+  - config-kafka-broker-consumer.properties
+- config-kafka-channel-data-plane
+  - config-kafka-channel-producer.properties
+  - config-kafka-channel-consumer.properties
```

Comment:
This didn't render the way you wanted

<img width="358" height="192" alt="Image" src="https://github.com/user-attachments/assets/773207b4-b787-4eac-95db-cee680812ada" />